Add cloud storage functionality to portfolio

efrainmunoz100 opened the pull request:
- Updates index.html to include a user submission form for images, and a button to see user submissions
- Creates uploadServlet.java to upload user images to cloud storage
- Creates userUploadsServlet.java to show user submissions

********************************************************
<img width="666" alt="Screen Shot 2021-03-09 at 10 29 03 PM" src="https://user-images.githubusercontent.com/63368883/110587780-eeccb480-8128-11eb-8680-6c2fd71981bf.png">
********************************************************
<img width="677" alt="Screen Shot 2021-03-09 at 10 31 21 PM" src="https://user-images.githubusercontent.com/63368883/110587844-01df8480-8129-11eb-9398-7c2afeef2778.png">
********************************************************
<img width="479" alt="Screen Shot 2021-03-09 at 10 41 02 PM" src="https://user-images.githubusercontent.com/63368883/110587878-10c63700-8129-11eb-8feb-cd9d94b24b88.png">

renguyen commented on portfolio/src/main/java/com/google/sps/servlets/userUploadsServlet.java:
```diff
@@ -0,0 +1,39 @@
+        // Output <img> elements as HTML.
+        response.setContentType("text/html;");
+        for (Blob blob : blobs.iterateAll()) {
+        String imgTag = String.format("<img src=\"%s\" />", blob.getMediaLink());
+        response.getWriter().println(imgTag);
+        response.getWriter().println("<br>");
```
Indent contents of for loop?

efrainmunoz100 commented:
- changed servlet and class names to be in CamelCase
- moved project name declarations out from the function, and declared them as class constants
- changed URLs to follow the hyphenation convention instead of camelCase

renguyen commented on portfolio/src/main/java/com/google/sps/servlets/FetchServlet.java:
```diff
@@ -8,7 +8,7 @@
 
 /** Responds with text when a request is sent to the /fetch URL */
 @WebServlet ("/fetchServlet")
```
I think this URL and the one in JsonServlet should also be hyphenated, but I'll leave that for a future PR should you choose to address it

renguyen commented on portfolio/src/main/java/com/google/sps/servlets/UploadServlet.java:
```diff
@@ -0,0 +1,62 @@
+@MultipartConfig
+public class UploadServlet extends HttpServlet{
+
+    private static final String projectId = "emunoz-sps-spring21";
```
Constants are typically all uppercase and words are separated with underscores, like PROJECT_ID and BUCKET_NAME

renguyen reviewed (approved):
Thanks for cleaning up the code! Some more minor comments about constant names and the other servlets' URLs, but I'll approve for now and you can address them in the future if you have time!
